06-data-carpentry.Rmd

```diff
@@ -294,75 +294,60 @@ data(wb_ineq)
 Renaming data columns is a common task that can make writing code faster by using short, intuitive names. The **dplyr** function `rename()` makes this easy.
 
 ```{block, varname-block, style='rmdtip'}
-Note in this code block the variable name is surrounded by back-quotes (\`).
+Note in this code block the variable name is surrounded by back-quotes (`\`).
 This allows R to refer to column names that are non-standard.
 Note also the syntax:
-`rename` takes the data frame as the first object and then creates new variables by specifying `new_variable_name = original_name`.
+`rename()` takes the data frame as the first object and then creates new variables by specifying `new_variable_name = original_name`.
 ```
 
 ```{r, message=FALSE}
-library("dplyr")
+data("wb_ineq")
```

Comment:
Would you change to `data("wb_ineq", package = "efficient")` for consistency.

Reply:
Sure - I think there is a duplication. Will sort it out.